src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -68,27 +67,36 @@ def check_source_access
         return if remote_source?
 
         options = { use_source: false, follow_project_links: true, follow_multibuild: true }
-        src_package = Package.get_by_project_and_name(source_project_name, source_package_name, options)
 
-        raise Pundit::NotAuthorizedError unless PackagePolicy.new(@token.user, src_package).create_branch?
+        begin
+          src_package = Package.get_by_project_and_name(source_project_name, source_package_name, options)
+        rescue Package::UnknownObjectError => e
+          raise Token::Errors::CanNotBranchPackageNotFound, "Package #{source_project_name}/#{source_package_name} not found, it could not be branched: #{e.message}"
```

Comment:
`e.message` would already say this right? So people would see:

`Package home:edu/ctris not found, it could not be branched: Package not found: home:edu/ctris`

Reply:
Fixed.